source/releasenotes/2024-07-22-New-Relic-V2-API-Migration

```diff
@@ -0,0 +1,8 @@
+---
+title: "New Relic V2 API Migration"
+published_date: "2024-07-22"
+categories: 
+---
+On July 2nd, 2024, Pantheon completed the migration of our customer's New Relic accounts to begin using the New Relic V2 APIs:
+* This allows Pantheon to leverage New Relic's latest account management features to manage its New Relic accounts. 
+* We don't expect any customer impacts from this change other than better support of their Pantheon-managed New Relic accounts.
```

Comment:
@tyherbert is this change in recommendation related? /release-notes/2024/06/quicksilver-examples-new-relic

Reply:
If so, this sentence should change accordingly

Reply:
Hey @stevector, Good catch. This does require a script update. We found a bug in the Quicksilver script and updated this today. It would be great to add this here, updated below:

```suggestion
* Pantheon customers using Quicksilver scripts to add New Relic deployment markers will need to use the [latest version of the New Relic deployments](https://github.com/pantheon-systems/quicksilver-examples/blob/main/new_relic_deploy/README.md) Quicksilver script to add deployment markers in New Relic moving forward.
```